### PR TITLE
Landing page 1.0

### PR DESCRIPTION
**Pull request by BradleyLi**
Signed-off-by: Bradley <jizhe.li@ucalgary.cm>

### CODE REVIEW

**Comment by tim-macphail** [comment]
This is a bit funky because I merged it too early then had you create a new branch. But looking through your files everything looks good, I see you resolved all the conversations that Yunfan started, and now that #18 is in this should be all good.
One thing: please delete the comment in lines 7-9 in Landing.tsx

**Review by tim-macphail — APPROVED** [review]
@jaza-k has plans to do some refactoring but I'm happy to put this into main for the time being 😀

File: client/tailwind.config.js [hunk 1]
```diff
@@ -1,108 +1,21 @@
...
-        sans: ["ui-sans-serif", "Lexend", "system-ui"],
-        display: ["Poppins", "system-ui"],
-        body: ["Lexend", "system-ui"],
+        poppins: "'Poppins', sans-serif",
       },
-      colors: {
-        primary: {
-          100: "#FFFFFF",
-          99: "#FFFBFF",
-          95: "#FFECED",
```
Comment (cloudyyoung) [inline client/tailwind.config.js:15]: This branch probably has merge conflict with the main branch at some point, but the resolving here is bad.
This reverts every change from #18, removing all added colour palette and fonts.

**Review by jaza-k — CHANGES_REQUESTED** [review]
Agree with @cloudyyoung, conflicts weren't resolved properly here, undoing #18. I created a new branch ```jaza/fix-bradley-landing``` which addresses these conflicts for now and makes it compatible with main. Further refactoring will be done soon

**Comment by cloudyyoung** [comment]
Closing this as #36 merged

**Pull request closed without merging**